### PR TITLE
Added .travis.yml

### PR DESCRIPTION
**Pull request by puff-tw**
Implements #7

### CODE REVIEW

**Comment by JuanitoFatas** [comment]
Could you add a [status badge](http://docs.travis-ci.com/user/status-images/) to README right below the Ruby.tw Reboot header? Thanks!

File: README.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
 # Ruby.tw Reboot
+[![Build Status](https://travis-ci.org/travis-ci/travis-web.png?branch=master)](https://travis-ci.org/travis-ci/travis-web)
```
Comment (JuanitoFatas) [inline README.md]: `travis-web.png` Shall we use svg? :blush:

Comment (puff-tw) [inline README.md]: Sorry, updated.

[Build Status](https://travis-ci.org/travis-ci/travis-api.svg)

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  - ruby-head
+
+matrix:
+  allow_failures:
+    - rvm: ruby-head
+  fast_finish: true
+
+notifications:
+  email: false
+  slack: rails-rookie-camp:2yVZEEVuEO9ZPQHwkvASnCRA
```
Comment (JuanitoFatas) [inline .travis.yml]: Update this to new slack and encrypt it :bow:

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+addons:
```
Comment (JuanitoFatas) [inline .travis.yml]: Do we need this addon?

File: .travis.yml [hunk 4]
```diff
@@ -0,0 +1,19 @@
+language: ruby
+sudo: false
```
Comment (JuanitoFatas) [inline .travis.yml:2]: Could you also add below this line:

``` yaml
cache:
  bundler: true
  bundler_args: --jobs=3 --retry=3 --without development
```

Read more about it at [here](http://docs.travis-ci.com/user/caching/).

Comment (puff-tw) [inline .travis.yml:2]: Done, ready to merge.

**Comment by JuanitoFatas** [comment]
@rubytaiwan/rubyists Please take a look! Leave :+1: or LGTM if you think this Pull Request is good to go.

File: .travis.yml (hunk 3 above)
Comment (puff-tw) [inline .travis.yml]: Remove Addons.

**Comment by JuanitoFatas** [comment]
I've committed your changes to master. Thanks for your contributions :+1:

**Pull request closed without merging**